src/20241106_AaveV3Ethereum_SafetyModuleStkAAVEReEnableRewards/SafetyModuleStkAAVEReEnableRewards.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Safety Module stkAAVE - Re-enable Rewards"
+author: "@karpatkey_TokenLogic"
+discussions: ""
+snapshot: "Direct-to-AIP"
+---
+
+## Simple Summary
+
+This publication proposes renewing the AAVE emissions to stkAAVE holders for the following 180 days.
+
+## Motivation
+
+With the conclusion of the recent reward cycle, this proposal aims to renew $AAVE emissions for stkAAVE holders. The stkAAVE module has proven to be a crucial mechanism as an AAVE supply sink, currently holding a TVL of $480M and accounting for 18.8% of the supply. It is recommended to maintain these emissions in anticipation of upcoming Umbrella developments, with adjustments to be made following the outcomes of the Umbrella upgrade.
+
+More information on stkAAVE can be found in these dashboards, [here](https://dune.com/xmc2/aave-safety-module) and [here](https://dune.com/KARTOD/AAVE-Staking).
+
+## Specification
+
+This proposal will implement the following changes to the AAVE emissions across the three SM categories:
+
+| Asset   | Current AAVE/Day | Proposed AAVE/Day |
+| ------- | ---------------- | ----------------- |
+| stkAAVE | 360              | 360               |
+
+The distribution end for 180 days from Execution date.
```

Comment:
@efecarranza The distributionEnd cannot be changed in stkAave, it is set as an immutable

Reply:
removed

Reply:
I think the changes were not pushed 😅